src/amo/components/SearchResult/index.js

```diff
@@ -19,15 +19,22 @@ export class SearchResultBase extends React.Component {
   static propTypes = {
     addon: PropTypes.object,
     i18n: PropTypes.object.isRequired,
+    showMetadata: PropTypes.bool,
+    showSummary: PropTypes.bool,
```

Comment:
Neato... though couldn't we just do this with CSS? I don't think it hurts to render the content.

Reply:
I guess if we NEVER want to render it this makes more sense, but it feels like something that could be done easier with CSS...